### PR TITLE
PP-6472 Persist and return gateway_payout_id on transaction

### PR DESCRIPTION
**Pull request by stephencdaly**
Add gateway_payout_id column to TransactionEntity and persist it as a
top-level field.

Return gateway_payout_id as field on transactions returned by the get
transactions API endpoints.

### CODE REVIEW

File: src/main/java/uk/gov/pay/ledger/transaction/model/Payment.java [hunk 1]
```diff
@@ -85,6 +85,7 @@ public Payment(Long id, String gatewayAccountId, Long amount,
         this.live = live;
         this.source = source;
         this.walletType = walletType;
+        this.gatewayTransactionId = gatewayTransactionId;
```
Comment (sfount) [inline src/main/java/uk/gov/pay/ledger/transaction/model/Payment.java:88]: I think this was intending to set the newly added payout id
```suggestion
        this.gatewayPayoutId = gatewayPayoutId;
```

Comment (stephencdaly) [reply src/main/java/uk/gov/pay/ledger/transaction/model/Payment.java:88]: ah dunno why i did this, it's set in the base class constructor so this line just needs to go

Comment (stephencdaly) [reply src/main/java/uk/gov/pay/ledger/transaction/model/Payment.java]: fixed

Comment (sfount) [reply src/main/java/uk/gov/pay/ledger/transaction/model/Payment.java]: Yeah much simpler 👍 looks good

**Pull request merged**